src/main/java/org/motechproject/mots/service/IvrService.java

```diff
@@ -180,6 +183,27 @@ public void saveCallDetailRecord(CallDetailRecord callDetailRecord,
     }
   }
 
+  /**
+   * Manually send Voto call log to Mots, this should be used only when call log is incorrect and
+   * must be manually changed before it can be successfully processed in Mots.
+   * @param votoCallLogDto it contains information about module progress (which trees was
```

Comment:
change "trees was changed" to "trees were changed" and "answers was chosen" to "answers were chosen"